client/server/server.js

```diff
@@ -40,13 +41,22 @@ connection.onInitialize((_params) => {
 documents.onDidChangeContent((change) => {
```

Comment:
All these methods combining “on”, the “to do” verb and “change” (another verb) make it sound as if it was a “did” / “do” use to put an emphasis over something that _did_ actually happen. Weird. Why not just `onChange(d)—`?

Reply:
Ask Microsoft :smile: